samples/dash-if-reference-player/app/main.js

```diff
@@ -244,6 +247,8 @@ app.controller('DashController', ['$scope', '$window', 'sources', 'contributors'
         autoLoad: false,
         muted: false,
         drmToday: false,
+        enableRollUp: true,
```

Comment:
We should not need these fields here as these two attributes are defined in `Settings.js`.

Reply:
Thanks, removed in 67a98bb.